tests/CMakeLists.txt

```diff
@@ -443,7 +443,7 @@ if(WITH_PYTHON)
   add_fdb_test(TEST_FILES slow/ParallelRestoreNewBackupCorrectnessAtomicOp.toml)
   add_fdb_test(TEST_FILES slow/ParallelRestoreNewBackupCorrectnessCycle.toml)
   add_fdb_test(TEST_FILES slow/ParallelRestoreNewBackupCorrectnessMultiCycles.toml)
-  add_fdb_test(TEST_FILES slow/ParallelRestoreNewBackupWriteDuringReadAtomicRestore.toml)
+  add_fdb_test(TEST_FILES slow/ParallelRestoreNewBackupWriteDuringReadAtomicRestore.toml IGNORE)
```

Comment:
Arguably we could disable all of the parallel restore tests, though we could wait to do so until we see additional problems

Reply:
Maybe we can just disable one unless the others are also blocking the release. By then, we can disable them and know which specific command to reproduce a failure.